fft-opt1.cu

```diff
@@ -148,6 +151,13 @@ int main() {
     // Copy result array from device to host
     cudaMemcpy(h_rev,d_rev,ARRAY_BYTES,cudaMemcpyDeviceToHost);
 
+    // Writing output to files
+    fprintf(fptr, "i\t\ta.real\t\t\ta.img\n");
+    for (int i = 0; i < ARRAY_SIZE; i++)
+    {
+        fprintf(fptr,"%d\t\t%f\t\t%f\n", i, h_rev[i].x, h_rev[i].y);
+    }
+
```

Comment:
let there be a column with the magnitude first and then the real and imaginary parts, i.e, the output format should be "i, |h_rev[i]|, h_rev[i].x, h_rev[i].y"